templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -1,5 +1,12 @@
 {% load search_tags %}
 
+{% if not has_enrichment_aggs_entries %}
+<!-- this is an example include to illustrate the no results message for tag view -->
+{% include "search/blocks/search_results_tag_no_results.html" %}
+<!--end comment-->
+{% endif %}
```

Comment:
The message seems not appearing correctly within the chart container as per design.  
Does this bit of code need to be somewhere inside `tag-frequency` container?

http://127.0.0.1:8000/search/catalogue/?q=qwert&vis_view=tag&chart_data_type=&group=community
![image](https://github.com/user-attachments/assets/81c66368-b0c7-4596-bf2c-79dd098cea70)